### ISSUE TITLE
Unicode "tag" characters are deprecated for language tagging

### ISSUE THREAD

**Issue by aphillips** [w3c/i18n-activity#1395]
**This is a tracker issue.** Only discuss things here if they are i18n WG internal meta-discussions about the issue. **Contribute to the actual discussion at the following link:**

§ https://github.com/w3c/webauthn/issues/1642

**Comment by xfq** [comment]
We can add a link to the `i18n` namespace: https://www.w3.org/TR/2020/REC-json-ld11-20200716/#the-i18n-namespace

BTW, `i18n` seems to be lowercase?

**Comment by r12a** [comment]
We should also point them to https://www.w3.org/TR/string-meta/#unicodeTags, since they seem to have looked at string-meta but not registered the advice not to use it.

Actually, we should probably also add some mustard saying don't do this.

**Comment by xorgy** [comment]
Is there any danger of this mechanism being truly broken? It seems that while the consortium has all sorts of negative things to say about the scheme, it is without meaningful direct alternatives. There is no direct alternative for in-band language tagging in plain text, and it also doesn't conflict with other non-deprecated uses, in applications that are aware of the scheme. The deprecation of the Unicode in-band language tagging scheme reads more like a ‘considered harmful’ than a recognizable set of real problems.

TL;DR: the reasons for deprecating this are overblown, and don't really have any impact on the particular use case; the scheme is easy to implement and hard to mess up, and isn't going to become broken.

**Comment by r12a** [comment]
@xorgy please add your comment to https://github.com/w3c/webauthn/issues/1642

This thread is for tracking that issue only, not for discussion outside the i18n WG.  People in the webauthn community will not see your comments here.
